src/main/java/nl/tudelft/semgroup4/collision/DefaultCollisionHandler.java

```diff
@@ -230,4 +248,34 @@ public void run() {
         Game.LOGGER.log(LogSeverity.DEBUG, "Collision", "Player picked up a utility");
         util.activate(game.getCurLevel());
     };
+    
+    private final CollisionHandler<HorMovingWall, AbstractWall> horWallWallHandler = 
+            (game, movingWall, abstractWall) -> {
+        int offset = 5;        
+        
+        if (movingWall.getLocX() < abstractWall.getLocX() 
```

Comment:
How about some method in AbstractGameObject that calculates this off-set based on the object's speed (this should be implemented at that level then as well). This way you can easily retrieve a working off-set for these purposes. Not sure that this is prettier, but at least this logic then does not pollute the higher-level code such as this code in the collisionHandler (where it is only interesting what happens, and we do not want to be distracted by how that happens).

< logic, organisation